sw/device/examples/sram_program/BUILD

```diff
@@ -178,7 +178,7 @@ OTP_IMAGES = [
         rom_bitstream = ":rom_otp_{}_exec_disabled".format(otp_name),
         rom_kind = "Rom",
         tags = [
```

Comment:
Should this be a targets argument instead of a tag?

Reply:
I think `targets` is a parameter of `opentitan_functest`, right?

Stepping back, if the purpose of the `cw310_rom` and `cw310_test_rom` tags was to reduce bitstream swapping in [ci/scripts/run-fpga-cw310-tests.sh](https://cs.opensource.google/opentitan/opentitan/+/master:ci/scripts/run-fpga-cw310-tests.sh;l=35;drc=46b156a66e1d0e4f7e651b55790347a8fae4fbc0), perhaps these tests deserve a new tag, e.g. `cw310_other`.

Reply:
I'm unfamiliar with this rule, but that sounds correct.

Reply:
SGTM until we find a better solution. You are both right, these are normally passed in the `targets` parameter of `opentitan_functest` but then get copied over to tags (see [here](https://cs.opensource.google/opentitan/opentitan/+/master:rules/opentitan_test.bzl;drc=7550a3512290c3c4df978fcc70112b787c963076;l=315)). We need to change `tags` here for the same effect.

Reply:
I think at your suggestino @dmcardle we're going to use cw310_rom_variant instead of cw310_other. This suggests the possibility of a spliced cw310_test_rom_variant, though none of those exist yet and they're not yet supported.